### ISSUE TITLE
How to use count(*)?

### ISSUE THREAD

**Issue by kun1988** [jeremychone/rust-sqlb#1]
code:
```
let sb = sqlb::select()
        .table("test")
        .columns(&["count(*)"]);
    println!("{}", sb.sql());
```

console:
`SELECT "count(*)" FROM "task"`

this sql ERROR:  column "count(id)" does not exist

**Comment by kun1988** [comment]
and can't add limit,offset

**Comment by jeremychone** [comment]
@kun1988 good ones. Let me work on this. Will be back soon. There should be a way to do the count with `raw` but could be good to have a `.count`. I will show both. 

Also, the `.limit()` and `.offset()` must be added. Will do this today or tomorrow. Feel free to ping me if I forget.

**Comment by jeremychone** [comment]
@kun1988 New version available, v0.3.3
- #4 new features - offset/limit
- #5 new feature - count

So, for the count, that should work: 

```rust
let sb = sqlb::select()
        .table("test")
        .columns(&["count(*)"]);
    println!("{}", sb.sql());
```

You can look at the `tests/` folders for more examples. 

Thanks for the feedback. They were spot on.
